app/services/edge_cache/bust/fastly.rb

```diff
@@ -23,7 +21,7 @@ def self.urls(path)
           URL.url("#{path}?i=i"),
         ]
       end
-      private_class_method :fastly_purge
+      private_class_method :urls
```

Comment:
Curious what this change is for?

Reply:
this `private_class_method` was referring a method that already has `private_class_method`. Seeing it's right below `urls` i imagine this is for that

Reply:
Oh... I see it now. Gotcha.